src/answers-umd.js

```diff
@@ -236,20 +237,32 @@ class Answers {
 
     this._onReady = parsedConfig.onReady || function () {};
 
+    const callPonyfillCssVariables = (callback) => {
```

Comment:
I think we should give this a different name and make it an actual function. I generally feel like anything over a couple of lines shouldn't be an inline function. Maybe we could name the function handlePonyfillCssVariables or something like that? Using the prefix 'call' makes it seem like we will always call the ponyfill. Depending on the value of disableCssVariablesPonyfill, we may or may not.

Reply:
good call!